services/docker-compose.yml

```diff
@@ -200,6 +200,7 @@ services:
   dask-sidecar:
     image: ${DOCKER_REGISTRY:-itisfoundation}/dask-sidecar:${DOCKER_IMAGE_TAG:-latest}
     init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}"
```

Comment:
why not adding as well the ``Task.Slot``  as well, i.e. ``"{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"``?
I am aware that there is a dask-sidecar per host at a time but it would allow us to discriminate with the closed ones (due to failure or shutdown)

Reply:
actually I discovered that a global service does not have the Task.Slot well defined...